### DISCUSSION TITLE
Any plan to upgrade Font Awesome icon-set to v6.0?

### DISCUSSION THREAD

**Discussion by ohmykreee** [Lednerb/bilberry-hugo-theme#358] [Ideas]
The new and official version of Font Awesome was just launched, adding some new icons (For me the most useful new icon is `fa-bilibili`).

So I'd like to ask if there is any plan to upgrade it to v6.0? Thanks.

**Comment by igor-baiborodine** [comment]
Let me check if we can upgrade to v6.0.

**Comment by metbril** [comment]
Have you considered migrating to the open source [ForkAwesome](https://forkaweso.me/Fork-Awesome/)?

**Reply by igor-baiborodine** [reply]
What would be the advantages of using ForkAwesome instead of FontAwesome?

**Reply by metbril** [reply]
For starters, ForkAwesome maintains the V4 way the icons are used, so no `fas-` for regular icons and `fab-` for brands. (Might be a personal preference.)

Additionally, it is community maintained. So it should be easier to have icons added.

**Comment by metamec** [comment]
Font Awesome V5 is really showing its age now new platforms are becoming established (Bluesky, Threads, etc.).

**Reply by igor-baiborodine** [reply]
@metamec, Bilberry **v4** was upgraded to using Font Awesome **V6**. Currently, there are no plans to use alternative icon providers.

**Reply by metamec** [reply]
> @metamec, Bilberry **v4** was upgraded to using Font Awesome **V6**. Currently, there are no plans to use alternative icon providers.

Oh that's great!  Thanks for telling me.  In hugo.toml (Bilberry V4) it still says "# define your links with FontAwesome 5 (only free icons are supported)".  I should have checked the FontAwesome folder before posting.
